app/templates/components/public/ticket-list.hbs

```diff
@@ -1,81 +1,87 @@
 <div class="ui segment">
   <h3 class="ui header">{{t 'Tickets'}}</h3>
 </div>
-<div class="ui segment">
-  <table class="ui ui very basic compact table" style="margin-bottom: 0">
-    <thead class="full-width">
-      <tr>
-        <th>{{t 'Type'}}</th>
-        <th class="four wide">{{t 'Sales Ends'}}</th>
-        <th class="ui right aligned two wide">{{t 'Price'}}</th>
-        <th class="one wide">{{t 'Quantity'}}</th>
-        <th class="ui right aligned two wide">{{t 'Subtotal'}}</th>
-      </tr>
-    </thead>
-    <tbody>
-      {{#each tickets as |ticket|}}
+<form class="ui segment form">
```

Comment:
Let's try to follow correct markup. Don't consider the form to be a div. Even though the above code is similar to the below in-terms of the output and working, but, we should always make form do what is is meant to do. 

```
<div class="ui segment">
<form class="ui form"> </form>
</div>
```